testing/correctness/tests/cli_tests/Makefile

```diff
@@ -0,0 +1,54 @@
+# include root makefile
+ifndef ROOT_MAKEFILE_MK
+include ../../../../Makefile
+endif
+
+# prevent rules from being evaluated/included multiple times
+ifndef $(abspath $(lastword $(MAKEFILE_LIST)))_MK
+$(abspath $(lastword $(MAKEFILE_LIST)))_MK := 1
+
+# The following are control variables that determine what logic from `rules.mk` is enabled
+
+# `true`/`false` to enable/disable the actual unit test command so it can be overridden (the targets are still created)
+# applies to both the pony and elixir test targets
+$(abspath $(lastword $(MAKEFILE_LIST)))_UNIT_TEST_COMMAND := false
+
+# `true`/`false` to enable/disable generate pony related targets (build/test/clean) for pony sources in this directory
+# otherwise targets only get created if there are pony sources (*.pony) in this directory.
+$(abspath $(lastword $(MAKEFILE_LIST)))_PONY_TARGET := false
+
+# `true`/`false` to enable/disable generate final file build target using ponyc command for the pony build target so
+# it can be overridden manually
+$(abspath $(lastword $(MAKEFILE_LIST)))_PONYC_TARGET := false
+
+# `true`/`false` to enable/disable generate exs related targets (build/test/clean) for elixir sources in this directory
+# otherwise targets only get created if there are elixir sources (*.exs) in this directory.
+$(abspath $(lastword $(MAKEFILE_LIST)))_EXS_TARGET := false
+
+# `true`/`false` to enable/disable generate docker related targets (build/push) for a Dockerfile in this directory
+# otherwise targets only get created if there is a Dockerfile in this directory
+$(abspath $(lastword $(MAKEFILE_LIST)))_DOCKER_TARGET := false
+
+# `true`/`false` to enable/disable recursing into Makefiles of subdirectories if they exist
+# (and by recursion every makefile in the tree that is referenced)
+$(abspath $(lastword $(MAKEFILE_LIST)))_RECURSE_SUBMAKEFILES := false
+
+CLI_PATH := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+CUSTOM_PATH += :$(EXTERNAL_SENDER_PATH)
+CUSTOM_PYTHONPATH += :$(DUMMY_PYTHON_PATH):../../../tools/integration/
```

Comment:
you shouldn't need to add `../../../tools/integration/` here, it's already included globally in `rules.mk`

Reply:
I know! But this was the only way I could get both the dummy app and the integration stuff working. Something's not quite right with `rules.mk`, and no matter what magic paths I put into CUSTOM_PYTHONPATH, I end up only being able to launch `alphabet` and `sequence_window` with machida.
Also, those two apps seem to be always available, regardless of Make variables defined in cli_tests/Makefile :\

Reply:
Ah, the place to fix this is here: https://github.com/WallarooLabs/wallaroo/blob/master/rules.mk#L53
Sequence Window and Alphabet (testing) shouldn't really be placed in the default path anymore.